server/require.js

```diff
@@ -55,9 +55,18 @@ export function getPagePath (page, {dir, dist}) {
 
 export default function requirePage (page, {dir, dist}) {
   const pagePath = getPagePath(page, {dir, dist})
+
   try {
     return require(pagePath)
   } catch (err) {
-    throw pageNotFoundError(page)
+    console.error(err)
```

Comment:
This should be under the `MODULE_NOT_FOUND` check, because otherwise *every* page that doesn't exist will be shot into the console. This includes things like favicon.ico and robots.txt in development.